src/azure-cli-tests/azure/cli/tests/exceptions.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+class CliTestError(Exception):
+    def __init__(self, error_message):
+        super(CliTestError, self).__init__(error_message)
+
+
+class CliExecutionError(Exception):
+    def __init__(self, exception):
+        self.exception = exception
+        message = 'An exception thrown during CLI execution indicates the command will fail. The ' \
+                  'exception was designed to be swallowed. Exception: [{}] {}.'
```

Comment:
I don't understand this message.  What is the reason for having CliTestError and CliExecutionError?